application/build.gradle.kts

```diff
@@ -3,5 +3,8 @@ dependencies {
     with(libs) {
         implementation(ktor.auth)
         implementation(ktor.auth.jwt)
+        implementation(avro)
+        implementation(kernel.presentation)
+        implementation(kernel.domain)
```

Comment:
Is it possible to move the dependency towards `kernel-domain` in the `domain` module and `kernel-avro` + `kernel-presentation` in the `presentation` one?

Reply:
no it is not possible because interfaces present in presentation and domain are required